cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -73,10 +73,7 @@
   SCHEDULE_NAME("schedule-name"),
   SCHEDULE_PROPERTIES("schedule-properties"),
   CONCURRENCY("concurrency"),
-  /**
-   * stream format
-   */
-  FORMAT("format"),
+
```

Comment:
unintended newline?

Reply:
I keep a newline because the following three properties are likely stream/view related, hence they are grouped together. I need more time to validate before those can be removed as well.